test/extensions/secondorder/diag_ggn/diag_ggn_settings.py

```diff
@@ -147,6 +148,32 @@
         "target_fn": lambda: regression_targets((3, 4 * 1 * 3 * 3)),
     },
 ]
+###############################################################################
+#                               Embedding                                     #
+###############################################################################
+LOCAL_SETTINGS += [
+    {
+        "input_fn": lambda: randint(0, 5, (6,)),
+        "module_fn": lambda: Sequential(
+            Embedding(5, 3),
+            Linear(3, 4),
+        ),
+        "loss_function_fn": lambda: CrossEntropyLoss(reduction="mean"),
+        "target_fn": lambda: classification_targets((6,), 4),
+    },
+    {
+        "input_fn": lambda: randint(0, 3, (3, 2, 2)),
+        "module_fn": lambda: Sequential(
+            Embedding(3, 2),
+            Flatten(),
+        ),
+        "loss_function_fn": lambda: CrossEntropyLoss(reduction="mean"),
+        "target_fn": lambda: classification_targets((3,), 2 * 2),
+        "seed": 1,
```

Comment:
can this line be removed?

Reply:
No, is necessary.

Reply:
Reason? Are the tolerances of `allclose` to strict with `seed=0`?

Reply:
from seeds 0-9 three seeds fail: 0, 4, 9. For all of them, the `diag_ggn_mc[_batch]_light` test on cuda fails. For seed 0 additionally the `diag_ggn_mc_batch_light` test on cpu fails.
I'm not sure what the reason is. The values are off by approximately 10%-30%.

Reply:
Could you quickly verify if these seeds still fail if you
- append a zero to `mc_samples` [here](https://github.com/tjs111/backpack/blob/5abe431b8f093e7b87643a50f9989ae4a9918196/test/extensions/secondorder/diag_ggn/test_diag_ggn.py#L50)
- append a zero to both `mc_samples` and `chunks` [here](https://github.com/tjs111/backpack/blob/5abe431b8f093e7b87643a50f9989ae4a9918196/test/extensions/secondorder/diag_ggn/test_diag_ggn.py#L72-L73)
(maybe you have to do the same for `BatchDiagGGNMC`

Reply:
Adding a zero to `mc_samples` ([here](https://github.com/schaefertim/backpack/blob/5abe431b8f093e7b87643a50f9989ae4a9918196/test/extensions/secondorder/diag_ggn/test_diag_ggn.py#L50) and [here](https://github.com/schaefertim/backpack/blob/5abe431b8f093e7b87643a50f9989ae4a9918196/test/extensions/secondorder/diag_ggn/test_batch_diag_ggn.py#L50)) would resolve this.

Reply:
Fine, then `0, 4, 9` are just 'unlucky' seeds. Thanks for double checking. Resolving.